### PR TITLE
t: Use more concise one-line Selenium driver initialization; Completely remove unused schema_hook functionality

### PR DESCRIPTION
**Pull request by okurz**
* t: Replace all schema hooks by more simple schema updates
* t: Use more concise one-line Selenium driver initialization
* t: Completely remove unused schema_hook functionality

### CODE REVIEW

**Review by kalikiana — COMMENTED** [review]
Maybe the test improvements deserve their own PR separated from the general fixes to how Selenium tests are initialized?

**Comment by okurz** [comment]
this is a followup after https://github.com/os-autoinst/openQA/pull/3403 so I marked it as draft

**Review by kalikiana — APPROVED** [review]

File: t/ui/15-admin-workers.t [hunk 1]
```diff
@@ -32,6 +32,8 @@ my $broken_worker_id  = 5;
 my $online_worker_id  = 6;
 my $offline_worker_id = 8;
 
+my $undef_var;
+
```
Comment (Martchus) [inline t/ui/15-admin-workers.t:36]: This variable doesn't seem to be used after all.

**Review by Martchus — CHANGES_REQUESTED** [review]

File: t/ui/15-admin-workers.t (hunk 1 above)
Comment (okurz) [reply t/ui/15-admin-workers.t:36]: Interesting. Who put that in :D

Comment (okurz) [reply t/ui/15-admin-workers.t]: fixed

**Review by Martchus — APPROVED** [review]

**Pull request merged**